gcp/table_gcp_kms_key_version.go

```diff
@@ -268,7 +268,8 @@ func kmsKeyVersionTurbotData(_ context.Context, d *transform.TransformData) (int
 		"Project":          project,
 		"Location":         location,
 		"KeyRing":          key_ring_name,
-		"ResourceName":     resource_name,
+		"KeyName":          resource_name,
```

Comment:
```suggestion
		"KeyName":          key_name,
```

On line 264, would it be better to rename the var this code uses to better reflect what it stores?